extra_data/export.py

```diff
@@ -216,7 +217,15 @@ def feed(self, data, metadata=None):
         self._buffer.put(self._serialize(data, metadata))
 
 
-def serve_files(path, port, source_glob='*', key_glob='*', **kwargs):
+def gen_time():
+    """Create a mock timestamp for meta-data"""
+    timestamp = time()
+    sec, frac = str(timestamp).split('.')
+    frac = frac.ljust(18, '0') # attosecond resolution
+    return sec, frac
+
+
+def serve_files(path, port, data_like='file', source_glob='*', key_glob='*', **kwargs):
```

Comment:
can you add the new keyword argument at the end, to keep order intact?

Reply:
Done